tests/remote/remote_controller.pm

```diff
@@ -17,39 +17,51 @@ use testapi;
 use utils;
 use mm_network;
 use lockapi;
+use mmapi;
 
 sub run {
-    my ($self) = @_;
     my $target_ip;
+    my $children = get_children();
+    my $child_id = (keys %$children)[0];
+    my $password = 'nots3cr3t';
```

Comment:
You can use type_password here - https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1250